lib/datadog/notifications/plugins/grape.rb

```diff
@@ -1,6 +1,6 @@
 module Datadog::Notifications::Plugins
   class Grape < Base
-    DEFAULT_EXCEPTION_HANDLER = ->(e) { self.class.exception_status(e) }
+    DEFAULT_EXCEPTION_HANDLER = ->(e) { Grape.exception_status(e) }
```

Comment:
Or, well, not as I think of it...

Maybe it's really worth the:

```ruby
def initialize(exception_status: nil)
  @exception_status = exception_status || -> (e) {...}
end
```

To capture proper class.

If you inline proc with `self.class` into arg declaration - that works for parent (caller) class.

But current `Grape...` solution doesn't care about subclasses.

Reply:
@dim still ^^